grails-app/services/com/unifina/service/UserService.groovy

```diff
@@ -189,15 +171,14 @@ class UserService {
 	}
 
 	SecUser getUserFromUsernameAndPassword(String username, String password) throws InvalidUsernameAndPasswordException {
-		PasswordEncoder encoder = new BCryptPasswordEncoder()
 		SecUser user = SecUser.findByUsername(username)
 		if (user == null) {
 			throw new InvalidUsernameAndPasswordException("Invalid username or password")
 		}
-		String dbHash = user.password
-		if (encoder.matches(password, dbHash)) {
+		String dbHash = user.password // TODO: $2a$10$z0HZdlGT7tvG6TSw4r/3Z.kqxJO4yM/ON4zX1pQ4TR1Kj3aidO/6q
```

Comment:
What's this TODO?

Reply:
I was testing password encryption and forgot about the comment.